packages/web-components/fast-components/src/accordion/README.md

```diff
@@ -0,0 +1,4 @@
+# fast-accordion
+`fast-accordion` is a web component implementation of an [Accordion](https://w3c.github.io/aria-practices/#accordion).
+
+For more information view the [component specification](./accordion.spec.md).
```

Comment:
Can you move the file here and add a checkmark to the spec roadmap. You'll likely need to update the pathing in the spec readme as well.

Reply:
Already moved it just didn't update the spec roadmap